docs/Serverless.md

```diff
@@ -0,0 +1,82 @@
+<h1 align="center">Serverless</h1>
+
+Run serverless applications and REST APIs using your existing fastify application.
```

Comment:
All cases of "fastify" in this document, as in this sentence, should be "Fastify".

Reply:
should be renamed now